### ISSUE TITLE
Dependency wheel in polar chart mode draws only half of the chart

### ISSUE THREAD

**Issue by ollivihervuori** [highcharts/highcharts#20850]
Chart settings where the issue replicates:
```
Highcharts.chart('container',{
	"chart": {
		"type": "dependencywheel",
		"polar": true
	},
	"series": [
		{
			"data": [
				{
					"from": "A",
					"to": "B",
					"weight": 1
				}
			]
		}
	]
});
```

#### Expected behaviour
I'm not sure how the dependency wheel in polar chart mode could be drawn, but the way how it's currently drawn seems illogical.

#### Actual behaviour
Only half of the chart is shown:
![image](https://github.com/highcharts/highcharts/assets/158522230/5f9abef1-d8b2-4aa6-98d5-a032880ea955)

#### Live demo with steps to reproduce
https://jsfiddle.net/wuLnrgv5/

#### Product version
v11.4.0

#### Affected browser(s)
At least Chrome, but issue is likely not browser related.

**Comment by karolkolodziej** [comment]
Hi @ollivihervuori!

I agree it looks strange, but I do not know how the dependency wheel could work with a polar chart.
Do you have any use case and idea of how it might work?

**Comment by ollivihervuori** [comment]
Yeah, the dependency wheel is already a polar kind of chart, so perhaps it cannot be converted into polar coordinates in a meaningful way. I believe the pie chart just ignores the polar checkbox, so perhaps the dependency wheel could work similarly.

We have a software allowing users to set the chart type and whether it's a polar chart, so our users ended up seeing this strange dependency wheel. We will solve the issue by disabling the polar for the dependency wheel, so this is not a critical issue for us.

**Comment by karolkolodziej** [comment]
Thank you @ollivihervuori for the explanation! 

I agree that the dependency wheel is already a polar kind of chart 😉 
Let's ignore the `polar` property for this series.
